test/EndToEndTests/Tests/Core/Microsoft.OData.Core.E2E.Tests/WriteJsonPayloadTests/SerializationInfoEdgeTests.cs

```diff
@@ -0,0 +1,488 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="SerializationInfoEdgeTests.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.OData;
+using Microsoft.AspNetCore.OData.Routing.Controllers;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OData.E2E.TestCommon;
+using Microsoft.OData.E2E.TestCommon.Common;
+using Microsoft.OData.E2E.TestCommon.Common.Server.EndToEnd;
+using Microsoft.OData.Edm;
+
+namespace Microsoft.OData.Core.E2E.Tests.WriteJsonPayloadTests
+{
+    public class SerializationInfoEdgeTests : EndToEndTestBase<SerializationInfoEdgeTests.TestsStartup>

```

Comment:
Consider adding a doc comment explaining what these tests are about. It's not clear just from the name what the focus area for these tests are.